Swiftfin/Views/SettingsView/UserDashboardView/ActiveSessionsView/Components/ActiveSessionRow.swift

```diff
@@ -38,29 +35,47 @@ extension ActiveSessionsView {
         private var rowLeading: some View {
             // TODO: better handling for different poster types
             Group {
-                if session.nowPlayingItem?.type == .audio {
+                switch session.nowPlayingItem {
+                case .none:
                     ZStack {
-                        Color.clear
-
-                        ImageView(session.nowPlayingItem?.squareImageSources(maxWidth: 60) ?? [])
-                            .failure {
-                                SystemImageContentView(systemName: session.nowPlayingItem?.systemImage)
-                            }
-                    }
-                    .squarePosterStyle()
-                } else {
-                    ZStack {
-                        Color.clear
-
-                        ImageView(session.nowPlayingItem?.portraitImageSources(maxWidth: 60) ?? [])
-                            .failure {
-                                SystemImageContentView(systemName: session.nowPlayingItem?.systemImage)
-                            }
+                        DeviceType(
+                            client: session.client,
+                            deviceName: session.deviceName
+                        ).clientColor
+
+                        Image(DeviceType(client: session.client, deviceName: session.deviceName).systemImage)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 40)
                     }
                     .posterStyle(.portrait)
```

Comment:
I may want these as square instead.

Reply:
This is what that would look like:
<img width="336" alt="Screenshot 2024-10-15 at 13 19 37" src="https://github.com/user-attachments/assets/25e8acbc-814f-49aa-982e-bf8e61806d12">

I originally had it as a square so I can go either way. Lemme know if you like this route and I'll leave this in the next commit!

Reply:
I think I like the square better. Just apply the `squarePosterStyle` to it so that we can get matched styling.